### PR TITLE
fix(navbar): revamp style and texts

### PR DESCRIPTION
**Pull request by FredericEspiau**
## Revamp Navbar
MBE-188

Related to:
- ovh-ux/ovh-manager-web#872
- ovh-ux/ovh-manager-dedicated#877
- ovh-ux/ovh-manager-cloud#1237

### CODE REVIEW

**Comment by jleveugle** [comment]
Hello @FredericEspiau,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci#1160.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report build-manager#1169.1 ✘  
* Stage 1
  * Build dependencies and inject Sentry ✘

File: client/app/app.less [hunk 1]
```diff
@@ -160,6 +159,7 @@
 @import './telecom/telephony/service/fax/campaigns/read/telecom-telephony-service-fax-campaigns-read.less';
 @import './telecom/telephony/service/fax/settings/telecom-telephony-service-fax-settings.less';
 @import './telecom/telephony/telecom-telephony.less';
+@import '../components/navbar/navbar.less';
```
Comment (cbourgois) [inline client/app/app.less]: We have something similar in https://github.com/ovh-ux/ovh-manager-telecom/pull/1214/files#diff-04bb1bd0b58a9a33f45862987152ac1fR127 : 
```less
@import './common/less/navbar.less';
```

**Comment by jleveugle** [comment]
Hello @FredericEspiau,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request merged**